### DISCUSSION TITLE
miss files:json_vc71_libmtd.lib when using vcpkg to install jsoncpp package

### DISCUSSION THREAD

**Discussion by PenNineCat** [microsoft/vcpkg#41961] [Q&A] [answered]
When I use vcpkg command"./vcpkg install jsoncpp" to instal jsoncpp ,I find the package misses json_vc71_libmtd.lib and my visual studio project cant find it .I try to rebuild it but it still misses the file.
Is a error by vcpkg or my mistake?

**Comment by dg0yt** [comment]
Why do you expect this filename to come from port jsoncpp?

**Reply by PenNineCat** [reply, marked as answer]
Sorry,maybe the project I copied doesnt use vcpkg to install jsoncpp.On my own project I can use jsoncpp without error.
